CTFd/challenges.py

```diff
@@ -36,7 +36,7 @@ def chals():
             if view_after_ctf():
                 pass
             else:
-                return redirect('/')
+                return redirect(url_for('/'))
```

Comment:
But that seems too uwsgi specific, when you could also deploy with gunicorn or Tornado or just the Flask server